SECURITY.md

```diff
@@ -0,0 +1,14 @@
+# Security Policy
+
+## Supported Versions
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 1.0.x   | :white_check_mark: |
+| < 1.0   | :x:                |
+
+## Reporting a Vulnerability
+
+Contact @Quantalabs at `npm.quantalabs@gmail.com`. **Do not** post an issue or any other public information about to vulnerability, 
```

Comment:
@quantum9Innovation will head the security for DiffPriv; we should discuss changing the contact info